frontend/src/components/layouts/Navbar.tsx

```diff
@@ -115,9 +116,9 @@ export const Navbar = ({ isVendorNavbar }: { isVendorNavbar: boolean }) => {
   if (!mounted || !mode) {
     return (
       <AppBar
-        position={isVendorNavbar ? "fixed" : "static"}
```

Comment:
did we need "fixed" for something vendor specific?

Reply:
Ah was this for https://github.com/klee97/directory-app/pull/160#discussion_r2593817833?

Reply:
The spacing should be fixed in my PR!